bluemix/authentication/iam/iam.go

```diff
@@ -306,6 +306,8 @@ func (c *client) doRequest(r *rest.Request, respV interface{}) error {
 			case "":
 			case "BXNIM0407E":
 				return authentication.NewInvalidTokenError(apiErr.errorMessage())
+			case "BXNIM0408E":
+				return authentication.NewRefreshTokenExpiryError(api.errorMessage())
 			case "BXNIM0400E":
```

Comment:
I think `api.errorMessage()` should be `apiErr.errorMessage()`

Reply:
@weieigao Thanks for pointing out. I've pushed a fix.